apps/server/src/modules/user-import/uc/user-import.uc.ts

```diff
@@ -56,7 +56,7 @@ export class UserImportUc {
 		const permissions = [UserImportPermissions.SCHOOL_IMPORT_USERS_UPDATE];
 		this.permissionService.checkUserHasAllSchoolPermissions(currentUser, permissions);
 
-		const userMatch = await this.userRepo.findById(userMatchId);
+		const userMatch = await this.userRepo.findById(userMatchId, true);
```

Comment:
what is true?

Reply:
rocket science :)

Reply:
was looking at wrong repo (via github) :D